jax_md/energy.py

```diff
@@ -891,6 +893,285 @@ def compute_fn(R, neighbor, **kwargs):
   return neighbor_fn, compute_fn
 
 
+# Tersoff model
+
+
+def load_lammps_tersoff_parameters(file: TextIO) -> Array:
+  """Reads Tersoff parameters from a LAMMPS file and returns parameter tables.
+
+  This function reads multi-element original Tersoff potential parameters
+  from a file.
+  
+  Args:
+    file: File handle for the Tersoff parameters text file.
+
+  Returns:
+    params: ndarray that contains Tersoff parameters.
+  """
+  # start to read files
+  # todo: params_per_line becomes input variables 
+  #       depending on the various type of tersoff model
+  params = []
+  params_per_line = 17
+  raw_text = file.read().split('\n')
+  file.close()
+
+  # read parameters 
+  # skip if the line has \# or empty
+  # if the number of parameters in one line is less than params_per_line,
+  # additional line is appended to match.
+  is_end = False
```

Comment:
Perhaps something like this could be a little cleaner: https://pastebin.com/CJgeVRFL

Reply:
Thanks very much. I changed them you suggested.